lib/auth/authProvider.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
```

Comment:
To be consistent with the rest of the codebase, this file (and mongoCredentials) should be named in `snake-case.js`